app/stac_api/models.py

```diff
@@ -230,13 +232,15 @@ def update_extent(self, item_properties_datetime):
             if self.extent["temporal"]["interval"][0][0] is None:
                 self.extent["temporal"]["interval"][0][0] = item_properties_datetime
                 self.save()
-            elif item_properties_datetime < self.extent["temporal"]["interval"][0][0]:
+            elif item_properties_datetime.replace(tzinfo=None) < datetime.strptime(
```

Comment:
I would not remove the tzinfo, but add it to the extent datetime. I would uses timezone aware times it makes usually life easier. From the spec the datetime should be displayed as UTC using iso format with timezone info from my understanding,